src/base/utils/misc.cpp

```diff
@@ -358,12 +358,20 @@ QString Utils::Misc::friendlyUnit(qint64 bytesValue, bool isSpeed)
     if (unit == SizeUnit::Byte)
         ret = QString::number(bytesValue) + " " + unitString(unit);
     else
-        ret = Utils::String::fromDouble(friendlyVal, 1) + " " + unitString(unit);
+        ret = Utils::String::fromDouble(friendlyVal, friendlyUnitPrecision(unit)) + " " + unitString(unit);
```

Comment:
Thanks @LordNyriox. It never crossed my mind that it's a special character.
I tried changing it locally but git wouldn't detect the change, thankfully github's online editor did.